### DISCUSSION TITLE
Next Release Date

### DISCUSSION THREAD

**Discussion by wsugarman** [fo-dicom/fo-dicom#1306] [Q&A] [answered]
I saw that 5.0.1 was deprecated on nuget.org due to critical bugs, and I was wondering when the next version would release? We'd love to leverage some of the features introduced in 5.0.1.

**Comment by smithago** [comment]
Particulary DicomFile.OpenAsync(stream, FileReadOption.ReadLargeOnDemand, 1000), where we can set the largeObjectSize. Also makes our dependencies cleaner.

**Comment by gofal** [comment, marked as answer]
ja, sure. Will do an release this week.

**Reply by smithago** [reply]
Thank you

**Comment by gofal** [comment]
5.0.2 is available via nuget

**Reply by wsugarman** [reply]
Thank you very much!
